app/models/foreman_tasks/task.rb

```diff
@@ -255,6 +255,33 @@ def action_continuous_output
       main_action.continuous_output.raw_outputs
     end
 
+    def self.get_latest_tasks_by_resource_ids(label, resource_ids, resource_type)
+      tasks = arel_table
+      dynflow_tasks = self::DynflowTask.arel_table
```

Comment:
is the `self::` necessary?

Reply:
Probably not, but rubocop did not complain.
I am not fully sure, whether I need to differentiate between `dynflow_tasks` and `tasks` because they resolve the same SQL-table :thinking:

Reply:
Probably not

Reply:
Let's get rid of it, it seems to work just fine without it